Manito/Manito/Screens/Detail-Ing/DetailngCodebaseViewController.swift

```diff
@@ -0,0 +1,354 @@
+//
+//  DetailngCodebaseViewController.swift
+//  Manito
+//
+//  Created by creohwan on 2022/11/03.
+//
+
+import UIKit
+
+import SnapKit
+
+// FIXME: 스토리보드 삭제 후 클래스명 변경 요
+final class DetailingCodebaseViewController: BaseViewController {
+
+    // MARK: - property
+    
+    private lazy var titleLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 34)
+        return label
+    }()
+    private lazy var periodLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .grey002
+        label.font = .font(.regular, ofSize: 16)
+        return label
+    }()
+    private lazy var statusLabel: UILabel = {
+        var label = UILabel()
+        label.backgroundColor = .mainRed
+        label.layer.masksToBounds = true
+        label.layer.cornerRadius = 11
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 13)
+        label.textAlignment = .center
+        return label
+    }()
+    private var missionBackgroundView: UIView = {
+        var view = UIView()
+        view.backgroundColor = .darkGrey004
+        view.layer.borderWidth = 1
+        view.layer.cornerRadius = 10
+        view.layer.borderColor = UIColor.systemYellow.cgColor
+        return view
+    }()
+    private let missionTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "오늘의 개별 미션"
+        label.textColor = .grey002
+        label.font = .font(.regular, ofSize: 14)
+        return label
+    }()
+    private lazy var missionContentsLabel: UILabel = {
+        var label = UILabel()
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 18)
+        return label
+    }()
+    private let informationTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "진행 정보"
+        label.textColor = .white
+        label.font = .font(.regular, ofSize: 16)
+        return label
+    }()
+    private lazy var manitteeBackView: UIView = {
+        let view = UIView()
+        let tapGesture = UITapGestureRecognizer(target: self, action: #selector(didTappedManittee))
+        view.addGestureRecognizer(tapGesture)
+        view.backgroundColor = .darkGrey002
+        view.makeBorderLayer(color: .white)
+        return view
+    }()
+    private let manitteeImageView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .subOrange
+        view.layer.cornerRadius = 49.5
+        return view
+    }()
+    private var manitteeIconView: UIImageView = {
+        var imageView = UIImageView()
+        imageView.image = ImageLiterals.icManiTti
```

Comment:
p5;

```suggestion
        let imageView = UIImageView(image: ImageLiterals.icManiTti)
```

이렇게도 쓸 수 있을 거 같네요.

Reply:
> p5;
> 
> 이렇게도 쓸 수 있을 거 같네요.

꿀팁 감사합니다! a77ba33